### PR TITLE
Refactor #3542: remove outdated javascript attribute

### LINKED ISSUE #3542: Remove outdated tag code.
### Describe the bug

According to this comment https://github.com/openemr/openemr/pull/3497#discussion_r424760810, In our codebase  have a lot of outdated code.

### Expected behavior
Remove and replace it.

Please assign this issue to me.

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #3542

#### Short description of what this resolves:
Remove outdated javascript attributes. e.g. `type="text/javascript"` `language="javascript"`

### CODE REVIEW

File: interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
 [-* we want to include our stylesheet for this view*-]
 <link rel="stylesheet" href="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery-datetimepicker/build/jquery.datetimepicker.min.css">
 
-<!--<script type="text/javascript" src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery/dist/jquery.min.js"></script>-->
-<script type="text/javascript" src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery-datetimepicker/build/jquery.datetimepicker.full.min.js"></script>
+<!--<script src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery/dist/jquery.min.js"></script>-->
+<script src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery-datetimepicker/build/jquery.datetimepicker.full.min.js"></script>
```
Comment (stu01509) [inline interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html:9]: Hello @bradymiller and @tywrenn 

Can I remove this comment script?

Comment (tywrenn) [reply interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html:9]: @stu01509 Go ahead since it's commented out

Comment (stu01509) [reply interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html]: @tywrenn Updated the commit.

**Comment by bradymiller** [comment]
hi @stu01509 , This looks good and bringing into the codebase. Thanks for the code improvements! -brady

I also made a video of my code review :)
**Warning: it is very dramatic. if you have any cardiac risk factors, i recommend not watching this video:**
https://www.loom.com/share/58ee615b7fb6436e88ff9c1473664444

**Pull request merged**